Exiled.API/Features/Door.cs

```diff
@@ -291,6 +291,21 @@ public static void LockAll(float duration, IEnumerable<ZoneType> zoneTypes, Door
                 LockAll(duration, zone, lockType);
         }
 
+        /// <summary>
+        /// Locks all <see cref="Door">doors</see> in the facility.
+        /// </summary>
+        /// <param name="duration">The duration of the lockdown.</param>
+        /// <param name="lockType">DoorLockType of the lockdown.</param>
```

Comment:
cref the `DoorLockType` without necessarily repeating "_of the lockdown_".

Reply:
> cref the `DoorLockType` without necessarily repeating "_of the lockdown_".

Just `` /// <param name="lockType"><see cref="DoorLockType"/>.</param>``?

Reply:
`<param name="lockType">The specified <see cref="DoorLockType"/>.</param>` should be ok.